odxtools/templates/macros/printAudience.xml.jinja2

```diff
@@ -19,32 +19,22 @@
 {%- endmacro -%}
 
 {%- macro printAudience(audience) -%}
-<AUDIENCE 
- {%- filter odxtools_collapse_xml_attribute %}
-  {%- if not audience.is_supplier %}
-                 IS-SUPPLIER="false"
+<AUDIENCE
+  {%- if audience._is_supplier is not none %}
+                 IS-SUPPLIER="{{bool_to_str(audience._is_supplier)}}"
```

Comment:
just a suggestion, maybe make a function that also handles None, and only print attribute if not None
```
print_bool_attr("IS-SUPPLIER", audience._is_supplier) 
```

Reply:
done.